### DISCUSSION TITLE
Referral System For Fluxstore Woocommerce

### DISCUSSION THREAD

**Discussion by Rana-Biswas** [inspireui/support#5612] [New Feature/Ideas Requests]
Can you please add a referral system to the app? Like if a customer buys a product worth or above 10$ then they get a 1$ coupon automatically in their account and if they refer to someone and the other person makes any purchase then also the first person gets a 1$ coupon which they can use to purchase items in the app.

**Comment by osman-kocak** [comment]
Its very good 

https://yithemes.com/themes/plugins/yith-woocommerce-affiliates/

**Reply by Rana-Biswas** [reply]
> Its very good
> 
> https://yithemes.com/themes/plugins/yith-woocommerce-affiliates/

I want this feature on mobile app with a separate banner on bottom of the home page

**Comment by osman-kocak** [comment]
Or 

https://codecanyon.net/item/sumo-affiliates-pro-wordpress-affiliate-plugin/22795996

Or

https://woocommerce.com/products/affiliate-for-woocommerce/
